e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc.yml

```diff
@@ -46,6 +46,44 @@ spec:
               value: "30100"
             - name: CLIENT_PORT_MAX
```

Comment:
Do you still use 1+2 image there?

Reply:
for now we do not remove support of our custom image (it will be done later)